src/Common/RJSF/rjsfForm.scss

```diff
@@ -9,10 +18,41 @@
 
     &__array-field-item {
         grid-template-columns: 1fr auto;
+        z-index: 0;
+        background: transparent;
 
         & > :first-child {
             // This is added as the array field itself has a margin bottom
             margin: 0;
         }
+
+        &:has(> .remove-btn__container:hover)::after {
+            background: var(--R50);
+        }
+
+        &:has(> .remove-btn__container)::after {
+            border-radius: 4px;
+            transform: translate(-50%, -50%);
+            top: 50%; 
+            left: 50%; 
+            background: transparent;
+            z-index: -1;
+            position: absolute;
+            height: calc(100% + 12px);
+            width: calc(100% + 12px);
```

Comment:
Fix the styling for Save button and footer for Catalog

Reply:
fixed and also added comments to demystify the magical 12px